Rakefile

```diff
@@ -0,0 +1,14 @@
+require 'data_mapper'
+require './lib/server'
+
+task :auto_upgrade do
+
+  DataMapper.auto_upgrade!
+  puts 'Auto-upgrade complete (no data loss)'
+end
+
+task :auto_migrate do
+
+  DataMapper.auto_migrate!
+  puts 'Auto-migrate complete (data could have been lost)'
+end
```

Comment:
It's good that you've extracted this out to a Rakefile, but then your README should explain that you need to run one of these tasks before you can run tests/start the server